packages/pages/src/App.js

```diff
@@ -21,8 +28,6 @@ const App = ({ effects, state }) => (
         Name
         <input
           type="text"
-          id="nameEntry"
-          name="name"
```

Comment:
This has nothing to do with this PR, I'll merge it but next time it should go in its own PR.